src/components/InteractionLayer/index.js

```diff
@@ -479,39 +434,17 @@ class InteractionLayer extends React.Component {
     }
   };
 
-  zoomed = () => {
-    const { ruler, zoomMode, onZoomXAxis } = this.props;
-    if (ruler && ruler.visible) {
-      this.processMouseMove(this.state.touchX, this.state.touchY);
```

Comment:
I believe this call is missed now, so the ruler does not stick to position while dragging.

Reply:
I can't figure out what this actually does -- I think you'll need to show me in-person.

Reply:
Done.